### PR TITLE
Update KMC machine configuration for GCP

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Update KMC machine configuration for GCP
- GCP machines [reference](https://cloud.google.com/compute/docs/general-purpose-machines#n2-standard)

**Related issue(s)**

### CODE REVIEW

File: components/kyma-metrics-collector/pkg/testing/fixtures/static_providers.json [hunk 1]
```diff
@@ -206,5 +206,23 @@
...
+    "n2-standard-32": {
+      "cpu_cores": 32,
+      "memory": 128
+    },
+    "n2-standard-64": {
+      "cpu_cores": 64,
+      "memory": 256
+    }
   }
 }
```
Comment (radufa) [inline components/kyma-metrics-collector/pkg/testing/fixtures/static_providers.json:228]: Please fix "nl"

File: resources/kcp/charts/kyma-metrics-collector/templates/public-cloud-info-config-map.yaml [hunk 2]
```diff
@@ -298,6 +298,44 @@ data:
...
+         "cpu_cores": 64,
+         "memory": 256
+       },
+       "n2-standard-80": {
+         "cpu_cores": 80,
+         "memory": 320
+       }
+     }
     }
 {{- end -}}
```
Comment (radufa) [inline resources/kcp/charts/kyma-metrics-collector/templates/public-cloud-info-config-map.yaml:341]: Here too

File: components/kyma-metrics-collector/pkg/testing/fixtures/static_providers.json (hunk 1 above)
Comment (mfaizanse) [reply components/kyma-metrics-collector/pkg/testing/fixtures/static_providers.json]: done

File: resources/kcp/charts/kyma-metrics-collector/templates/public-cloud-info-config-map.yaml (hunk 2 above)
Comment (mfaizanse) [reply resources/kcp/charts/kyma-metrics-collector/templates/public-cloud-info-config-map.yaml]: done

**Review by radufa — APPROVED** [review]

**Pull request merged**